test/example_packages/makefile_complex/fpm.toml

```diff
@@ -0,0 +1,8 @@
+name = "makefile_complex"
+
+[dependencies]
+with_makefile = { path = "../with_makefile" }
+
+[library]
+source-dir = "src"
```

Comment:
This is the default, isn't it? So it can be removed?

Reply:
Right now if you have a `[library]` section, it has to contain `source-dir`. I can make it optional, I just haven't done that yet.